### PR TITLE
Hugo: show inline code like parent

### PR DESCRIPTION
**Pull request by anne-usmedia**
- Inherit font size and line height from parent
- Always have a normal font style, eg no italic
- White code background when section is blue or grey

For
- https://linear.app/usmedia/issue/CUE-174
- https://linear.app/usmedia/issue/CUE-181

### CODE REVIEW

**Review by JorindeUsMedia — CHANGES_REQUESTED** [review]
Maybe not have a background when inline-code is inside toc & tree. 
FYI: checked on this page: https://deploy-preview-332--cue.netlify.app/docs/howto/encode-json-yaml-with-cue/

File: hugo/assets/scss/base/base.scss [hunk 1]
```diff
@@ -332,13 +332,19 @@ pre {
...
+    :is(dd, td, h1, h2, h3, h4, h5, h6, p, span, ul, ol) & {
         background-color: var(--pre-background-color);
         border-radius: $b-radius;
+        font-size: inherit;
+        font-style: normal;
+        line-height: inherit;
         padding: 3px 5px;
     }
+
+    .section--blue &,
```
[the remark this answers is not in the document]
Comment (JorindeUsMedia) [reply hugo/assets/scss/base/base.scss:344]: Nope its fine like this

**Review by myitcv — APPROVED** [review]
LGTM. I compared https://alpha.cuelang.org/docs/howto/encode-json-yaml-with-cue/ with https://deploy-preview-332--cue.netlify.app/docs/howto/encode-json-yaml-with-cue/ to verify.

Importing into Gerrit now.

**Comment by myitcv** [comment]
Closed via a11077d856676086962b9fa42d166f1de166de0f

**Pull request closed without merging**